### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by pushkar2112**
## Issue
Create an issue template #41

<br>

## Description

Add issue template to ISSUE_TEMPLATE folder in .github

<br>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation 

<br>

## Screenshot
not required

<br>

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] The app runs properly
- [x] I have performed a self-review of my own code
- [x] I have formatted and commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Issue_Template
```
Comment (Saransh-cpp) [inline .github/ISSUE_TEMPLATE/issue_template.md:2]:
```suggestion
name: Report an issue
```

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+about: Use this template to raise any issues or bugs that you come across in the project
+title: ''
+labels: ''
+assignees: ''
```
Comment (Saransh-cpp) [inline .github/ISSUE_TEMPLATE/issue_template.md:6]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+# Description
+Provide a little description for the issue
```
Comment (Saransh-cpp) [inline .github/ISSUE_TEMPLATE/issue_template.md:11]:
```suggestion
Provide a clear description of the issue
```

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
Thanks, @pushkar2112! Looks good! Suggestions -
- Never commit on master/main, always create a new branch.

A few minor comments -

**Comment by pushkar2112** [comment]
> * Never commit on master/main, always create a new branch.

Sry about that, I created a new branch but merged it in my forked repo.

> A few minor comments -

Pls check for the changes suggested.

**Review by Saransh-cpp — APPROVED** [review]
Thanks, @pushkar2112! Looks good! Which program are you a part of?

**Comment by Saransh-cpp** [comment]
@all-contributors please add @pushkar2112 for docs

**Comment by pushkar2112** [comment]
> Thanks, @pushkar2112! Looks good! Which program are you a part of?

Codepeak

**Pull request merged**